binderhub/templates/page.html

```diff
@@ -2,19 +2,41 @@
 <html>
 <head>
   <title>{% block title %}Binder{% endblock %}</title>
-  {% block head %}
+
+  {% block meta %}
+  {% endblock meta %}
+
+  {# Social media previews #}
+  {% block meta_social %}
+  <meta property="og:title" content="The Binder Project">
+  <meta property="og:image" content="https://mybinder.org/static/images/logo_social.png">
+  <meta property="og:description" content="Reproducible, sharable, open, interactive computing environments.">
+  <meta property="og:image:width" content="1334">
+  <meta property="og:image:height" content="700">
+  <meta property="og:image:alt" content="The Binder Project Logo" />
+  <meta name="twitter:card" content="summary_large_image">
+  {% endblock meta_social %}
+
+  {% block favicon %}
   <link id="favicon" rel="shortcut icon" type="image/png" href="{{static_url("favicon.ico")}}" />
-  <link href="{{static_url("dist/styles.css")}}" rel="stylesheet"></link>
+  {% endblock %}
+  {% block head %}
```

Comment:
meta inside head?

Reply:
this makes sense. but on the other hand nested blocks might be difficult when people want to customise. maybe better I should rename "head" block to "extra_head"?

Reply:
I changed my mind and made them nested in page.html.